clusterctl/clusterdeployer/clusterdeployer.go

```diff
@@ -366,11 +401,39 @@ func pivot(from, to ClusterClient) error {
 		cluster.SetResourceVersion("")
 		err = to.CreateClusterObject(cluster)
 		if err != nil {
-			return err
+			return fmt.Errorf("error moving Cluster '%v': %v", cluster.GetName(), err)
 		}
 		glog.Infof("Moved Cluster '%s'", cluster.GetName())
 	}
 
+	fromDeployments, err := from.GetMachineDeploymentObjects()
+	if err != nil {
+		return err
+	}
+	for _, deployment := range fromDeployments {
+		// New objects cannot have a specified resource version. Clear it out.
+		deployment.SetResourceVersion("")
+		err = to.CreateMachineDeploymentObjects([]*clusterv1.MachineDeployment{deployment})
```

Comment:
`if err := ...; err != nil { ... }`

Reply:
Changed -- also changed two other occurrences in this method to be consistent